manifests/0/0xGingi/Browser/114.0.5735.134/0xGingi.Browser.locale.zh-CN.yaml

```
@@ -0,0 +1,27 @@
+# Created with DuckDuckStudio's automation script.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.9.0.schema.json
+
+PackageIdentifier: 0xGingi.Browser
+PackageVersion: 114.0.5735.134
+PackageLocale: zh-CN
+Publisher: 0xGingi-Browser 作者
+PublisherUrl: https://0xgingi.com/
+PublisherSupportUrl: https://github.com/0xGingi/0xgingi-browser-windows/issues
+Author: 0xGingi
+PackageName: 0xGingi-Browser
+PackageUrl: https://github.com/0xGingi/0xgingi-browser-windows
+License: BSD-3-Clause license
+LicenseUrl: https://github.com/0xGingi/0xgingi-browser/blob/main/LICENSE
+Copyright: Copyright 2023 0xGingi-Browser作者. 保留所有权利.
+ShortDescription: 一个去Google化的Chromium浏览器
+Description: 一个实验性的去Google化Chromium分支，旨在增强隐私和安全性。
+Tags:
+- 浏览器
+- 去Google化Chromium
+ReleaseNotesUrl: https://github.com/0xGingi/0xgingi-browser-windows/releases/tag/114.0.5735.134
```

```suggestion
ReleaseNotesUrl: https://github.com/0xGingi/0xgingi-browser-windows/releases/tag/114.0.5735.134-1
```

Sorry, automation error.